src/utility/_ellipsis.scss

```
@@ -0,0 +1,29 @@
+/*doc
+---
+title: Ellipsis
+name: ellipsis
+category: Utility
+---
+
+文章がフォームからはみ出てしまう場合、文章を格納している要素にこのユーティリティを使用することで、
```

使うところがフォームとは限らないので、こんな感じかな

```suggestion
文章が要素からはみ出てしまう場合、このユーティリティを使用することで、
```